src/Factory.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: AGPL-3.0-or-later
+
+pragma solidity 0.8.17;
+
+import {DssVestMintable} from "./DssVest.sol";
+
+contract DssVestNaiveFactory {
+
+    function createDssVestMintable(address forwarder, address companyToken, address companyAdminAddress) external returns (address) {
```

Comment:
Looks good. But I still would feel better if we could find an existing audited smart contract that can be used for deployments.

Reply:
I totally agree. But I think such a contract does not exist, because each instance needs these specific arguments on creation.